packages/plugin-core/src/commands/NoteLookupCommand.ts

```diff
@@ -652,25 +671,19 @@ export class NoteLookupCommand
    * and when the date portion is the last bit of the hierarchy.
    * e.g.) if the picker value is journal.2021.08.13.some-stuff, we don't override (title is some-stuff)
    */
-  journalTitleOverride(): string | undefined {
-    if (this.isJournalButtonPressed()) {
-      const quickpick = this.controller.quickpick;
-
-      // note modifier value exists, and nothing else after that.
-      if (
-        quickpick.noteModifierValue &&
-        quickpick.value.split(quickpick.noteModifierValue).slice(-1)[0] === ""
-      ) {
-        const [, ...maybeDatePortion] = quickpick.noteModifierValue.split(".");
-        // we only override y.MM.dd
-        if (maybeDatePortion.length === 3) {
-          const maybeTitleOverride = maybeDatePortion.join("-");
-          if (maybeTitleOverride.match(/\d\d\d\d-\d\d-\d\d$/)) {
-            return maybeTitleOverride;
-          }
+
+  journalTitleOverride(input: string): string | undefined {
+    if (/.*\d{4}\.\d{2}\.\d{2}$/g.test(input)) {
```

Comment:
is there a better way to test for the input? this misses the case for example where format is `y.dd.MM`.

can we use luxon parsing for this? https://github.com/moment/luxon/blob/master/docs/parsing.md

Reply:
good point!